### ISSUE TITLE
Thumbnail placeholder not working for Whitehall assets

### ISSUE THREAD

**Issue by chrisroos** [alphagov/asset-manager#291]
Requesting a Whitehall image asset that's in the `unscanned` state will attempt to redirect you to a placeholder image. The URL of this placeholder image responds with a 404.

To replicate (in integration):

1. Either disable the Sidekiq workers or simply comment out the `after_save :schedule_virus_scan` line in asset.rb so that we can get an asset in the 'unscanned' state.
2. Find a Whitehall image asset in the asset-manager Rails console (e.g. `asset = WhitehallAsset.where(legacy_url_path: /jpg/).first`)
3. Update the asset's state to 'unscanned'
4. Request the asset using the `asset.legacy_url_path` (e.g. https://assets-origin.integration.publishing.service.gov.uk/government/uploads/system/uploads/organisation/logo/249/GOVUKlogo.jpg)
5. Observe that you're redirected to https://assets-origin.integration.publishing.service.gov.uk/images/thumbnail-placeholder.png and that visiting that URL results in a 404

NOTE that the thumbnail placeholder is actually available at https://assets-origin.integration.publishing.service.gov.uk/asset-manager/thumbnail-placeholder.png.

## Curl output of requesting an unscanned Whitehall image asset

```
$ curl https://assets-origin.integration.publishing.service.gov.uk/government/uploads/system/uploads/organisation/logo/249/GOVUKlogo.jpg -I -L

HTTP/1.1 302 Found
Server: nginx
Date: Tue, 07 Nov 2017 14:45:08 GMT
Content-Type: text/html; charset=utf-8
Location: https://assets-origin.integration.publishing.service.gov.uk/images/thumbnail-placeholder.png
Connection: close
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
Cache-Control: max-age=60, public
X-Request-Id: d6fbd62f-09ad-45c4-99fe-735f0866df9b
X-Runtime: 0.093166
Strict-Transport-Security: max-age=31536000
Access-Control-Allow-Origin: *
Access-Control-Allow-Methods: GET, OPTIONS
Access-Control-Allow-Headers: origin, authorization

HTTP/1.1 404 Not Found
Server: nginx
Date: Tue, 07 Nov 2017 14:45:08 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 0
Connection: close
Status: 404 Not Found
X-Request-Id: 22874885-5fd3-456e-aec2-2ea5b4053922
X-Runtime: 0.002584
```

**Comment by chrisroos** [comment]
I've opened PR #292 to address this problem.

**Comment by chrisroos** [comment]
PR #292 has been deployed so I need to check this behaviour on integration before closing this issue.

**Comment by chrisroos** [comment]
@chrislo and I have just confirmed that this is now working in integration.

We updated the asset with legacy URL "/government/uploads/system/uploads/organisation/logo/301/logo2.jpg" so that it was in the "unscanned" state. Requesting the asset redirected us to https://assets-origin.integration.publishing.service.gov.uk/asset-manager/thumbnail-placeholder-b0b645c0f44c2fc7d8cc2fee24f576d82256a85ac3c3e0a59d6ff60d2903d6b6.png as expected.
